3001-4000/LIT3311Prosph.xml

```diff
@@ -65,7 +65,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             </keywords>
          </textClass>
          <abstract>
-            <p>A letter (prosphoneticus) by <persName ref="PRS3255CyrilI"/> addressed to Princesses Arcadia and Marina, sisters of <persName ref="PRS9480Theodosi"/>. The letter, originally composed in Greek, discusses the nature of Christ, aiming to limit the influence of <persName ref="PRS7553Nestoriu"/>.
+            <p>A letter (<foreign xml:lang="lat">prosphoneticus</foreign>) by <persName ref="PRS3255CyrilI"/> addressed to Princesses Arcadia and Marina, sisters of <persName ref="PRS9480Theodosi"/>. The letter, originally composed in Greek, discusses the nature of Christ, aiming to limit the influence of <persName ref="PRS7553Nestoriu"/>.
```

Comment:
Is it `xml:lang="la"` or `xml:lang="lat"`? If it's `lat`, then I should update quite a number of BAV manuscripts, thank you @nafisa-valieva

Reply:
It is `la` https://betamasaheft.eu/Guidelines/?id=languageIntro

Reply:
@nafisa-valieva the section on `langUsage `should include a list of all language abbreviations used as `xml:lang` element in the record